test/unit/app/models/publishes_to_publishing_api_test.rb

```diff
@@ -101,23 +107,48 @@ def test_published_gone_handler; end
     test_object.publish_gone_to_publishing_api
   end
 
-  test "enqueues a worker to bulk republish the document on request" do
-    test_object = TestObject.new
-    class << test_object
-      include PublishesToPublishingApi
+  context "when object can be published to Publishing API" do
+    test "bulk_republish_async enqueues a worker to bulk republish the document" do
+      test_object = TestObject.new
+      class << test_object
+        include PublishesToPublishingApi
+      end
```

Comment:
Is it worth moving this to a `before` block like you do for the next context?

Reply:
Ooh, yes, I'll move that into a `before` to keep things consistent.